torch/jit/_recursive.py

```diff
@@ -186,6 +186,12 @@ def infer_type(name, item):
 
             continue
 
+        if callable(value):
+            builtin_symbol_name = _find_builtin(value)
```

Comment:
Nit: I think `callable()` is a weird check here. We should make _find_builtin to trivially return None if the thing is not callable instead of checking it in the outer scope (or it just won't find `value` in the table which is okay too).